docs/docs/index.md

```diff
@@ -30,31 +29,31 @@ More guides can be found in the [how-to guides](how-to/index.md) section.
 
 ## [References](references/index.md)
 
-There is plenty of knowledge to grasp in order to understand what Firefly III and the data importer are capable of, such as:
+Do you want to learn what **Firefly III** and the **Data Importer** are capable of? Check out the [references](references/index.md) section to find such information as:
 
-- [The Firefly III API](references/firefly-iii/api/index.md)
+- [How to use the Firefly III API](references/firefly-iii/api/index.md)
```

Comment:
This page doesn't tell you how to use the Firefly III API, I'd rather not call it a "how to".

Reply:
Ah, that's my mistake. I meant for it to link to docs/docs/how-to/firefly-iii/features/api.md. Also, now that I think about it, it'd probably have been better to word that as "How to talk to the API" since that's the main header in api.md is (unless this conflicts with the distinction between tutorials and how-to guides, in which case maybe the header in api.md should be reworded as well). Thoughts?

Reply:
Sounds good!

Reply:
Alright, I'll make an updated PR!